metadata-ingestion/src/datahub/ingestion/source/qlik_sense/qlik_sense.py

```diff
@@ -215,7 +217,9 @@ def _gen_dashboard_urn(self, dashboard_identifier: str) -> str:
             name=dashboard_identifier,
         )
 
-    def _gen_dashboard_info_workunit(self, sheet: Sheet) -> MetadataWorkUnit:
+    def _gen_dashboard_info_workunit(
+        self, sheet: Sheet, app_id: str
+    ) -> MetadataWorkUnit:
         dashboard_urn = self._gen_dashboard_urn(sheet.id)
```

Comment:
do we know if the `sheet.id` is globally unique?

Reply:
Public sheet id is unique among all objects present internally. Don't know if it is unique globally.
And private sheet id is not in UUID format. Eg: 'ULVLjqu'. Hence we can consider private sheet id as not unique globally.
https://community.qlik.com/t5/New-to-Qlik-Sense/Sheet-Id-from-QMC-vs-URL/td-p/1630337